HomeWork 5/LoggerAplication/LoggerAplication/Action.cs

```diff
@@ -0,0 +1,26 @@
+﻿namespace LoggerAplication
+{
+    internal class Action
+    {
+        public static Result StartMethod()
+        {
+            Logger.GetInstance().Log("Satrt Method", LogType.Info);
+
+            return new Result(true);
+        }
+
+        public static Result SkippedLogicInMethod()
+        {
+            Logger.GetInstance().Log("Skipped Logic In Method", LogType.Warning);
+
+            return new Result(true);
+        }
+
+        public static Result IBrokeALogic()
```

Comment:
Naming

Reply:
1 method “Start method:” and the name of this method.
2 method “Skipped logic in method:” and the name of this method
3 method The method returns Result where the error property = “I broke a logic”.
selected a name from the task))